meta/main.yml

```
@@ -4,17 +4,17 @@ galaxy_info:
   description: Elastic Repositories
   author: Thomas Widhalm <thomas.widhalm@netways.de>
   company: Netways GmbH
-  license: GPL-3.0-or-later
+  license: license(GPLv3)
```

```suggestion
  license: GPL-3.0-or-later
```

Ok, then the old version should work. Sorry, that was my bad suggesting in an offline discussion to remove the `-or-later` part because I suspected it being not compliant.

---

Might just be for Galaxy metadata https://docs.ansible.com/ansible/latest/dev_guide/collections_galaxy_meta.html#structure

Still it's probably good to keep the identifiers from spdx